### DISCUSSION TITLE
Missing game covers in Ubisoft source

### DISCUSSION THREAD

**Discussion by earstreet** [lutris/lutris#4704] [Q&A]
Hello!

I just installed Lutris on my new Steam Deck. Because of some problems with error 256 I did several completely new installations in the last days.
I noticed that most of the times the game covers in the Ubisoft source window are not complete. Mostly they are all blank.
What can I do to reload the covers?

**Comment by danieljohnson2** [comment]
Clicking the 'reload' button on the sidebar next to the service should download the missing media for that service- if they are available.
